### ISSUE TITLE
install broken when using cmake 3.16.3

### ISSUE THREAD

**Issue by udif** [MikePopoloski/slang#629]
**Describe the bug**
873fb228d1c64d1564c5a714cbafcbb08b51900b is breaking install on cmake 3.16.3

**To Reproduce**
After compiling slang, run `sudo make install`
The output is:
```
...
-- Install configuration: "Release"
CMake Error at cmake_install.cmake:51 (file):
  file INSTALL cannot find
  "/home/udif/git/slang/../external/ieee1800": No
  such file or directory.
```

**Additional context**
Please see comment in https://github.com/MikePopoloski/slang/commit/873fb228d1c64d1564c5a714cbafcbb08b51900b
If this is no longer supported then please fix 
`cmake_minimum_required(VERSION 3.15...3.24)`

**Comment by udif** [comment]
Upon further checks, I get the exact same failures even with cmake 3.24.2.

**Comment by MikePopoloski** [comment]
Fixed in 8809f2cc51dedcda350a22559c568344e6438695
